master/pkg/model/experiment.go

```diff
@@ -465,6 +465,28 @@ type TrialMetrics struct {
 	Metrics      JSONObj    `db:"metrics" json:"metrics"`
 }
 
+// Represent order of active states (Queued -> Pending -> Running).
+var allocationStateIndex = map[experimentv1.State]int{
```

Comment:
maybe this? this function already exists for allocation states
```suggestion
var experimentStateIndex = map[experimentv1.State]int{
```